### ISSUE TITLE
Add BTCPay Host Onion capability

### ISSUE THREAD

**Issue by MaxHillebrand** [btcpayserver/btcTransmuter#87]
Follow up to #40.

Currently I cannot set a v3 onion of a BTCPay in the `BTCPayserver External Service Data / BTCPay Host Url`. 

![screenshot_20210205_003500](https://user-images.githubusercontent.com/30683012/107018605-f18e5f80-67a0-11eb-9463-2d5f088a7913.png)

**Comment by Kixunil** [comment]
I believe you have to click that "Authorize Client" link first and follow the procedure.

**Comment by BitizenOne** [comment]
@MaxHillebrand Are you running BTCPay through docker?

**Comment by MaxHillebrand** [comment]
No, a deb package from https://github.com/debian-cryptoanarchy/cryptoanarchy-deb-repo-builder/

**Comment by BitizenOne** [comment]
Interesting, I'll have to check out that project. I'm not sure how the architecture works, but using a containerized btcpayserver, I was able to connect external services through tor using a similar technique that I used here: https://github.com/btcpayserver/btcTransmuter/issues/96#issuecomment-1083272261
